library/quality/lint-baseline.xml

```diff
@@ -1,76 +1,52 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<issues format="4" by="lint 2.3.0-alpha1">
+<issues format="4" by="lint 2.3.0-alpha2">
 
     <issue
-        id="UnusedIds"
-        message="The resource `R.id.facebook_button` appears to be unused"
-        errorLine1="    android:id=&quot;@+id/facebook_button&quot;/>
-"
-        errorLine2="    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
-        <location
-            file="src\main\res\layout\idp_button_facebook.xml"
-            line="5"
-            column="5"/>
-    </issue>
-
-    <issue
-        id="UnusedIds"
-        message="The resource `R.id.google_button` appears to be unused"
-        errorLine1="    android:id=&quot;@+id/google_button&quot;/>
-"
-        errorLine2="    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
-        <location
-            file="src\main\res\layout\idp_button_google.xml"
-            line="5"
-            column="5"/>
-    </issue>
-
-    <issue
-        id="UnusedIds"
-        message="The resource `R.id.twitter_button` appears to be unused"
-        errorLine1="    android:id=&quot;@+id/twitter_button&quot;/>
+        id="UnusedAttribute"
+        message="Attribute `transitionGroup` is only used in API level 21 and higher (current min is 16)"
```

Comment:
There's no way to backport this right? That's why I've added all these to the baseline.

Since I don't think I explained this before, the easiest way to update the baseline is to rename the file to something else and run lint. Once lint creates a new baseline, use Intellij's diffing tool (Ctrl+D) on the two baselines.